content/transfer-photos-from-phone.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Transferring photos from your phone to Pop!_OS
+description: >
+  How to move your photos from your Android or iPhone to Pop!_OS
+keywords:
+  - System76
+  - Apple
+  - OS X
+  - macOS
+  - Android
+  - Google
+  - switch
+  - PopOS
+  - Pop!\_OS
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: switching
+tableOfContents: true
+---
+
+## From Pixel/Android to Pop!_OS
+
+### Changing the USB Preference
+
+When you connect your Pixel/Android device to your system there is a notification about USB Preferences which can be reached when you swipe from the top of the screen like the screenshot below:
+
+![Pixel USB Notification](/images/transfer-photos-from-phone/Pixel-USB-Notification.png)
+
+Once you press on the notification the follow screen will appear:
+
+![Pixel USB Preferences](/images/transfer-photos-from-phone/Pixel-USB-Preferences-1.png)
+
+Simply change "no data transfer" to "File transfer / Android Auto":
+
+![Pixel USB Preferences](/images/transfer-photos-from-phone/Pixel-USB-Preferences-2.png)
+
+### Open the Files application
+
+Next open the <u>Files</u> application and right-click on the Pixel 8 item (in this example it may say the OEM of your device such as Samsung) in the menu on the left side of the window to access the file system on the device.
+
+![Pixel File System](/images/transfer-photos-from-phone/Pixel-Files-1.png)
+
+## From iPhone to Pop!_OS
+
+### Trusting the computer in iOS
+
+First the iPhone must trust your computer which then allows access to the file system on the iPhone. This can be done using the prompt that appears when you connect the iPhone to your system in this screenshot:
```

Comment:
I think we should either add a comma after "First," or remove it and start the sentence with "The"

Reply:
Would removing "in this screenshot" be cleaner?